scripts/type-generation/webdriver-generate-typings.js

```diff
@@ -15,8 +15,13 @@ for (const [protocolName, definition] of Object.entries(PROTOCOLS)) {
         for (const [, description] of Object.entries(methods)) {
             const { command, parameters = [], variables = [], returns } = description
             const vars = variables
-                .filter((v) => v.name != 'sessionId' && v.name != 'elementId')
-                .map((v) => `${v.name}: string`)
+                .filter((v) => v.name != 'sessionId')
+                .map((v) => {
+                    if (v.name === 'elementId') {
+                        return `${v.name}: ElementId`
+                    }
+                    return `${v.name}: string`
+                })
```

Comment:
could you add comments why you filter bu `sessionId` and have if block with condition `elementId`. I think it would be helpful for folks who, possibly, would improve it or change.

Reply:
hey @CrispusDH I'll add comment, however I've noticed that I made a mistake here.

Let me fix it :)